cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -44,6 +48,22 @@ const Modules = () => {
   useEffect(() => {
     setLoadingModules(true);
 
+    function populateNamespaceData() {
```

Comment:
Because we are using this logic only here, you don't have to create a function and call upon it later.

Just call the `axios.get...` straight from here and it should be good!